phpunit-multisite.xml.dist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
```

Comment:
`phpunit-multisite.xml` and `phpunit-multisite.xml.dist` need to be added to `.distignore`

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/2534/commits/2ea2e7ccf4b32ac09e1885de03c532878ebf60f0